### PR TITLE
[Mono.Android] Add a Test for reading 9 Patch files at runtime.

### PR DESCRIPTION
**Pull request by dellis1972**
Context https://bugzilla.xamarin.com/show_bug.cgi?id=46807

### CODE REVIEW

File: src/Mono.Android/Test/Resources/Resource.designer.cs [hunk 1]
```diff
@@ -81,6 +81,9 @@ public partial class Drawable
 			// aapt resource value: 0x7f020004
 			public const int Icon = 2130837508;
 			
+			// aapt resource value: 0x7f020005
+			public const int image = 2130837509;
```
Comment (jonpryor) [inline src/Mono.Android/Test/Resources/Resource.designer.cs]: Why is this `image` when the `@(AndroidResource)` is using `Image.9.png`? I'd expect this member to be `Image` (capital `I`).

**Comment by jonpryor** [comment]
build

**Comment by dellis1972** [comment]
build

**Comment by dellis1972** [comment]
build

**Comment by dellis1972** [comment]
@jonpryor this might be useful :D I think we all forgot about it lol

**Comment by dellis1972** [comment]
@jonpryor woot Green Build! :)

**Pull request merged**